---
title: "'Finally' não pode aparecer fora de uma instrução 'Try'"
ms.date: 07/20/2015
f1_keywords:
- vbc30382
- bc30382
helpviewer_keywords:
- BC30382
ms.assetid: 0314d8d2-18bc-4bbd-858c-0a18408b52fd
ms.openlocfilehash: 60f02c9000e99c6f1b91fb5f0e2929c4e696e549
ms.sourcegitcommit: f8c270376ed905f6a8896ce0fe25b4f4b38ff498
ms.translationtype: MT
ms.contentlocale: pt-BR
ms.lasthandoff: 06/04/2020
ms.locfileid: "84403726"
---
# <a name="finally-cannot-appear-outside-a-try-statement"></a>'Finally' não pode aparecer fora de uma instrução 'Try'
`Finally`é usado para concluir um `Try...Catch...Finally` bloco; portanto, ele só pode aparecer uma vez no final do bloco. Você tem um desnecessário `Finally` ou sua `Finally` instrução aparece fora dos limites de seu `Try` bloco correspondente.  
  
 **ID do erro:** BC30382  
  
## <a name="to-correct-this-error"></a>Para corrigir este erro  
  
1. Localize e remova a instrução desnecessária `Finally` .  
  
2. Mova a `Finally` instrução para o local apropriado em seu código.  
  
## <a name="see-also"></a>Confira também

- [Instrução Try...Catch...Finally](../language-reference/statements/try-catch-finally-statement.md)
